app/helpers/hyrax/work_form_helper.rb

```diff
@@ -6,8 +6,9 @@ module WorkFormHelper
     #   from views. refactor to avoid
     # @return  [Array<Array<String, String, Hash>] options for the admin set drop down.
     def admin_set_options
-      service = Hyrax::AdminSetService.new(controller)
+      return @admin_set_options.select_options if @admin_set_options
```

Comment:
With this PR, it seems like we're making this the default. So how would one fall
through this guard `return` and hit the older Service/Presenter setup below? Is it in the case where folks have overridden the controller downstream?

Reply:
any controller override would hit it, but i also think there's at least one other controller (DashboardCollections ?) that routes to this helper.

Reply:
i had been thinking to explicitly deprecate the fall-through path here once all the relevant internal controllers are populating `@admin_set_options`

Reply:
Ah, ok that's great. Thanks!